integration-tests/src/test/java/oracle/kubernetes/operator/utils/LoadBalancer.java

```diff
@@ -146,6 +146,7 @@ private void upgradeTraefikNamespace() throws Exception {
     StringBuffer cmd = new StringBuffer("helm upgrade ");
     cmd.append(" traefik-operator")
        .append(" stable/traefik ")
+       .append(" --debug ")
```

Comment:
Why are you adding the "--debug" flag here?  Is it because we are going to assert on debug output?

Reply:
I want to know the reason if the helm command fails so that it will be easier to debug when the test fails